File: api/health.js
// Vercel Serverless Function — /api/health
import { runHandler } from './_lib/respond.js';

export default async function handler(req, res) {
  await runHandler(req, res, async () => ({
    status: 'ok',
    service: 'home-agent',
  }));
}

File: api/health.py
# @vercel/python
"""Vercel Serverless Function — /api/health"""

from __future__ import annotations

from api._lib.base_handler import APIHandler
from api._lib.logic import NotFound


class handler(APIHandler):
    def route(self, method, path, query):
        if path == "/api/health" and method == "GET":
            return {"status": "ok", "service": "home-agent"}
        raise NotFound(f"no route: {method} {path}")
